partner_event/readme/CONTRIBUTORS.rst

```diff
@@ -6,3 +6,4 @@
 * David Vidal <david.vidal@tecnativa.com>
 * Rafael Blasco <rafael.blasco@tecnativa.com>
 * Anil Kesariya <anil.r.kesariya@gmail.com>
+* Víctor Martínez <victor.martinez@tecnativa.com>
```

Comment:
We have changed the way to reflect contributors here with:

- No emails, for avoiding spam
- Grouping all our contributors under a general point with company's website. Example: https://raw.githubusercontent.com/OCA/account-invoicing/13.0/account_invoice_validation_queued/readme/CONTRIBUTORS.rst

Reply:
The file has been changed.